src/react-app-env.d.ts

```diff
@@ -1 +1,23 @@
 /// <reference types="react-scripts" />
+
+interface Post {
+  id:number,
+  userId: number,
+  title: string,
+}
+
+interface User {
+  id:number,
+  username: string,
+  name: string,
+}
+
+// type Comment = {
+//   id: number,
+//   postId: number,
+//   name: string,
+//   email: string,
+//   body: string,
+//   createdAt?: Date,
+//   updatedAt?: Date,
+// };
```

Comment:
?

Reply:
it's weird but it was showing error somewhere in the code until i moved Comment type to another file...no idea why